dg/media/configs.js

```diff
@@ -1140,10 +1140,12 @@ function() {
 
     var misc = {
         download_chunk_size: 2000,
+        data_transfer_schema: "uploadqueue",
+        data_transfer_credentials_schema: "user",
```

Comment:
based on previous comments of health and nutrition i was under the impression that you might say fields are hardcoded that's why i have picked from configs.